sirepo/package_data/static/js/sirepo.js

```diff
@@ -1842,10 +1842,19 @@ SIREPO.app.factory('panelState', function(appState, requestSender, simulationQue
         if (optionIndex < 0) {
             throw new Error('no enum value found for ' + model + '.' + field + ' = ' + value);
         }
-        var opt = $(fieldClass(model, field)).find('option')[optionIndex];
+
+        let sel = $(fieldClass(model, field));
+        // apply to a specific instance
+        if (instance != null) {
+            sel = sel.eq(instance);
+        }
+        // apply to all instances
+        const f = i => i % eType.length === optionIndex;
+        let opt = sel.find('option').filter(f);
+
         if (! opt) {
             // handle case where enum is displayed as a button group rather than a select
-            opt = $(fieldClass(model, field)).find('button')[optionIndex];
+            opt = sel.find('button').filter(f);
         }
```

Comment:
This part doesn't seem to work the enums displayed as a button group. For an example, run the warpvnd Two-Poles example, then run the Field Calculation on the source tab, and the open the Field Comparison Report dialog, it should hide one of the dimensions since the simulation is 2d:

![x](https://github.com/radiasoft/sirepo/assets/6516910/e7f73fb7-d75d-47c4-9384-6b4f4adac190)

Reply:
[f6a08cd](https://github.com/radiasoft/sirepo/pull/6463/commits/f6a08cdb770b66adc56450af7da774f6d0100a30)